web_src/js/features/repo-settings.ts

```diff
@@ -122,7 +122,7 @@ function initRepoSettingsBranches() {
 function initRepoSettingsOptions() {
   if ($('.repository.settings.options').length > 0) {
     // Enable or select internal/external wiki system and issue tracker.
-    $('.enable-system').on('change', function () {
+    $('.enable-system').on('change', function (this: HTMLInputElement) {
```

Comment:
FYI this is a [fake parameter](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#specifying-the-type-of-this-for-functions).